fixtures/tests/test_fixture.py

```diff
@@ -162,7 +180,7 @@ def test_details_from_child_fixtures_are_returned(self):
             self.assertEqual({}, parent.getDetails())
             # After cleanup the child details are still gone.
             child.addDetail('foo', 'content')
-        self.assertEqual({}, parent.getDetails())
+        self.assertRaises(TypeError, parent.getDetails)
```

Comment:
getDetails tries to dereference self._details. So this is just checking things are indeed wiped out, and its not really a user interface.